test/libsolidity/syntaxTests/literalSuffixes/parameters/mantissa_out_of_range.sol

```diff
@@ -37,16 +37,16 @@ contract C {
 // TypeError 8838: (549-558): The type of the literal cannot be converted to the parameters of the suffix function.
 // TypeError 8838: (568-577): The type of the literal cannot be converted to the parameters of the suffix function.
```

Comment:
As the TODO above says, our goal is to improve all of these errors. We should make it clear to the user why exactly we cannot convert the value. Right now the message is clear only for the extreme cases that exceed `uint256`. We need to do better.

Looks like these cases currently get rejected because there's no implicit conversion to the smaller type. I think that just indicating in the current message what we're trying to convert to what type would be good enough.

This is especially important for fractional numbers because then the value is different from what user sees in the literal - e.g. you should say that mantissa is x, exponent is y and x or y cannot be converted to some type T.